src/frontend/packages/core/src/core/extension/extension-types.ts

```diff
@@ -2,6 +2,9 @@ import { Type } from '@angular/core';
 import { FormGroup } from '@angular/forms';
 import { Schema, schema } from 'normalizr';
 
+import { EndpointModel } from '../../../../store/src/types/endpoint.types';
+import { TableCellCustom } from '../../shared/components/list/list.types';
```

Comment:
The intent with the extensions folder/package was that it should not import anything else in the Stratos code base - thus it could be a separate package and extensions would only need to import the extensions package - they would not need to import anything else.

Is there a way to loosen the typing on listDetailsComponent and check it at runtime for compatibility?